appchain/src/lib.rs

```diff
@@ -1025,6 +1027,20 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		#[pallet::weight(<T as Config>::WeightInfo::delete_token_id())]
+		pub fn delete_token_id(origin: OriginFor<T>, token_id: Vec<u8>) -> DispatchResult {
```

Comment:
Do I need to call `destory` function in `pallet-assets` to destroy the created assets here?

Reply:
In substrate IBC, you may need. But not here.